### PR TITLE
Silence warnings when accessing user attributes. Fixes #190

### LINKED ISSUE #190: Future warnings in pydevd_resolver
## Environment data

- PTVSD version: master
- OS and version: Windows
- Python version (& distribution if applicable, e.g. Anaconda): 3.7.3
- Using VS Code or Visual Studio: VSC

## Actual behavior

```
C:\GIT\ptvsd\src\ptvsd\_vendored\pydevd\_pydevd_bundle\pydevd_resolver.py:214: FutureWarning: Series.base is deprecated and will be removed in a future version
  attr = getattr(var, name)
C:\GIT\ptvsd\src\ptvsd\_vendored\pydevd\_pydevd_bundle\pydevd_resolver.py:214: FutureWarning: Series.data is deprecated and will be removed in a future version    
  attr = getattr(var, name)
C:\GIT\ptvsd\src\ptvsd\_vendored\pydevd\_pydevd_bundle\pydevd_resolver.py:214: FutureWarning: Series.flags is deprecated and will be removed in a future version   
  attr = getattr(var, name)
C:\GIT\ptvsd\src\ptvsd\_vendored\pydevd\_pydevd_bundle\pydevd_resolver.py:214: FutureWarning: Series.itemsize is deprecated and will be removed in a future version
  attr = getattr(var, name)
C:\GIT\ptvsd\src\ptvsd\_vendored\pydevd\_pydevd_bundle\pydevd_resolver.py:214: FutureWarning: Series.strides is deprecated and will be removed in a future version
  attr = getattr(var, name)
```
## Expected  behavior

Should not show warnings.

## Steps to reproduce:
```py
import pandas as pd
df = pd.read_csv("https://data.nasa.gov/api/views/gh4g-9sfh/rows.csv?accessType=DOWNLOAD")
```
Expanding `df.GeoLocation` shows this warning.

### PR DESCRIPTION
**Pull request by fabioz**

### CODE REVIEW

File: src/debugpy/_vendored/pydevd/_pydev_bundle/pydev_console_utils.py [hunk 1]
```diff
@@ -447,6 +448,7 @@ def getFrame(self):
 
         return xml.getvalue()
 
+    @silence_warnings_decorator
```
Comment (fabioz) [inline src/debugpy/_vendored/pydevd/_pydev_bundle/pydev_console_utils.py:451]: Note that I'm silencing in the caller and not in the actual resolver to avoid having to start/stop to catch the warnings for each new variable.

File: src/debugpy/_vendored/pydevd/pydevd_attach_to_process/attach_pydevd.py [hunk 2]
```diff
@@ -38,6 +38,7 @@ def process_command_line(argv):
 
 
 def main(setup):
+    sys.path.append(os.path.dirname(__file__))
```
Comment (fabioz) [inline src/debugpy/_vendored/pydevd/pydevd_attach_to_process/attach_pydevd.py:41]: This is actually unrelated (it's an issue in the attach to process where under a particular scenario the `add_code_to_python_process` couldn't be imported).

**Review by int19h — APPROVED** [review]
